rosdep/python.yaml

```diff
@@ -4292,6 +4292,12 @@ python-selenium-pip:
   ubuntu:
     pip:
       packages: [selenium]
+python-semantic-version:
+  debian: [python-semantic-version]
+  fedora: [python2-semantic_version]
+  ubuntu:
+    bionic: [python-semantic-version]
+    xenial: [python-semantic-version]
```

Comment:
I first thought it's strange to explicitly have only 2 distro entries, then found this pkg of Py ver 2 is available only on these 2 distros https://packages.ubuntu.com/xenial/python-semantic-version. So +1.